src/abstract-chart.js

```diff
@@ -258,12 +266,12 @@ class AbstractChart extends Component {
         >
           <Stop
             offset="0"
-            stopColor={this.props.chartConfig.color()}
-            stopOpacity="0.1"
+            stopColor={fillShadowGradient}
+            stopOpacity={fillShadowGradientOpacity}
```

Comment:
it's kind of odd that you can set the first Stop opacity but not the second one's